src/Components/Information.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+ const Information = props => {
+    return (
+        <div className="info">
+            <p>{props.date}</p>
+            <p>{props.service}</p>
+            <p>{props.media}</p>
```

Comment:
So the this actually has another purpose. `media_type` checks whether the `url` is for an image or for a video, so we don't have to display that, but we can use it to conditionally render an image, or a video, depending on the provided url